ringpop-admin.js

```diff
@@ -38,6 +38,8 @@ function main() {
         .command('status', 'Status of members in ring')
         .command('partitions', 'Show partition information of a ring')
         .command('top', 'General membership information')
+        .command('reap', 'Remove the faulty nodes from this nodes membership')
+        .command('heal', 'Start a partition heal coordinated by the target node')
```

Comment:
I would just alter the description to: "attempt to heal a partition; might need to be invoked multiple times".

By design, we didn't put strong guarantees to this admin method, because, for the operator, it isn't really clear what to expect:
1. Does it heal all partitions? In what time?
2. What should it return if it heals one, but not the other?
3. Time out, if any?

We can create a "robust" version to `ringpop-admin` command, which checks for the global cluster state and invokes as many calls as required. However, I don't think it's very relevant, since healing is running by default periodically.